js/main.js

```diff
@@ -272,30 +272,74 @@ requirejs(['bootstrap'], function () { // bootstrap must come first
 			}
 		});
 
-		pageModel.loadIncluded = function () {
-			pageModel.loadingIncluded(true);
-			var includedPromise = $.Deferred();
-
-			$.ajax({
-				url: sharedState.vocabularyUrl() + 'lookup/identifiers',
+		pageModel.loadAncestors = function(ancestors, descendants) {
+			return $.ajax({
```

Comment:
Move such things to services (either find existing in WebAPIProvider or create a new one)